sleap/io/video.py

```diff
@@ -806,6 +806,9 @@ class SingleImageVideo:
     height_: Optional[int] = attr.ib(default=None)
     width_: Optional[int] = attr.ib(default=None)
     channels_: Optional[int] = attr.ib(default=None)
+    grayscale: Optional[bool] = attr.ib()
```

Comment:
I'm a little concerned this might break deserialization. This is because [cattrs](https://cattrs.readthedocs.io/en/latest/unstructuring.html#attrs-classes-and-dataclasses) tries to create instances by matching the attribute names in the class with the ones that are saved.

Do we have a test for loading files with `SingleImageVideo`s saved in the old format? (Creating one at runtime in the current version would save it with the `grayscale` attribute, so it wouldn't catch this case.)